en/cloud_development_setup/README.md

```diff
@@ -0,0 +1,5 @@
+# RunCode Cloud Environment setup
+
+> **Note** If you already worked through the [installation steps](../installation/README.md), no need to do this again – you can skip straight ahead to [Introduction to Python](../python_introduction/README.md).
```

Comment:
I would amend to "If you have already worked through the installation steps, you do not need to also complete the RunCode Cloud Environment setup. Please skip straight to Introduction to Python"